### PR TITLE
Add origin and license hints for pip code

### PR DESCRIPTION
**Pull request by pombredanne**
Signed-off-by: Philippe Ombredanne <pombredanne@nexb.com>

### CODE REVIEW

File: src/packaging_dists/_utils.py [hunk 1]
```diff
@@ -7,8 +28,11 @@ def _find_name_version_sep(fragment, canonical_name):
     # type: (str, str) -> int
     """Find the separator's index based on the package's canonical name.
 
-    Copied verbatim from pip. This function is needed since the canonicalized
-    name does not necessarily have the same length as the egg info's name part.
+    Copied verbatim from:
+      pip._internal.index.package_finder._find_name_version_sep
```
Comment (uranusjr) [inline src/packaging_dists/_utils.py:32]: Could you change this (and the one below) to use rst code blocks?

```
Copied verbatim from::

    pip._internal.index.package_finder._find_name_version_sep
```

Comment (pombredanne) [reply src/packaging_dists/_utils.py]: Done!

**Comment by uranusjr** [comment]
Thanks! (p.s. feel free to ping me if you need a new release. I’m not doing it since I don’t want to disrupt you sending in patches.)

**Comment by pombredanne** [comment]
> Thanks! (p.s. feel free to ping me if you need a new release. I’m not doing it since I don’t want to disrupt you sending in patches.)

A release would much welcomed at this stage!
Thank you ++

**Comment by uranusjr** [comment]
I’ve published 0.2 to PyPI.

**Comment by pombredanne** [comment]
You are a mensch! I am going to use this to roll my own actually working alternative to `pip download` that behaves too strictly. I will push this in a few!

**Pull request merged**